### DISCUSSION TITLE
CI/CD Workshop

### DISCUSSION THREAD

**Discussion by tauhid621** [github/india#265] [General]
Thread for discussions related to CI/CD workshop 2021.

Actions overview : [link](https://youtu.be/cP0I9w2coGU)
Workshop 1 : [link](https://bit.ly/ActionsWorkshop-1)

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
